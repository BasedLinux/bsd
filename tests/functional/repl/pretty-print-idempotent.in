:l pretty-print-idempotent.bsd
oneDeep
oneDeep
twoDeep
twoDeep
oneDeepList
oneDeepList
twoDeepList
twoDeepList
