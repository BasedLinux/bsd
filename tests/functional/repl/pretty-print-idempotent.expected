Bsd <bsd version>
Type :? for help.

bsd-repl> :l pretty-print-idempotent.bsd
Added <number omitted> variables.

bsd-repl> oneDeep
{ homepage = "https://example.com"; }

bsd-repl> oneDeep
{ homepage = "https://example.com"; }

bsd-repl> twoDeep
{
  layerOne = { ... };
}

bsd-repl> twoDeep
{
  layerOne = { ... };
}

bsd-repl> oneDeepList
[ "https://example.com" ]

bsd-repl> oneDeepList
[ "https://example.com" ]

bsd-repl> twoDeepList
[
  [ ... ]
]

bsd-repl> twoDeepList
[
  [ ... ]
]
